packages/table/src/stores/tableStore.ts

```diff
@@ -37,15 +37,14 @@ const useOverrideSizeFactory = (
     [setOverrides]
   );
 
-// jotai supports setting with functions, but createAtomStore doesn't know that
```

Comment:
This was true in earlier versions of `jotai-x`, but is now only true if `atom` is used explicitly when setting the initial state, and this is accurately reflected in the types.